### PR TITLE
add scrollable element config

### PR DESCRIPTION
**Pull request by lardern**
Add a configuration called `scrollable`,specifies the ancestor element that can be scrolled,default value is `$(window)`

### CODE REVIEW

File: table-fixed-header.js [hunk 1]
```diff
@@ -1,67 +1,79 @@
-(function($) {
+(function ($) {
 
-$.fn.fixedHeader = function (options) {
- var config = {
-   topOffset: 38
-   //bgColor: 'white'
- };
- if (options){ $.extend(config, options); }
+  $.fn.fixedHeader = function (options) {
```
Comment (oma) [inline table-fixed-header.js]: unneccessary diff on untouched code. Even if you have a preference on number of spaces, a good rule is to never do such formatting along with code changes, it masks the code changes. If done in separate commits it would be acceptable, even though the whole PR is more difficult to review. Please create PR of the code changes only, thanks for contributing.

Comment (lardern) [reply table-fixed-header.js]: Sorry,I didn't notice that.
